galaxyproject/galaxy
# Add sorting function to Saved Rules Selector

Adding sorting to SavedRuleSelector as requested in #13192 to improve usefulness/ease of finding rules

## How to test the changes?
(Select all options that apply)
  1. Create a few sets of rules in the Rule Builder (you can use the [Rule Based Uploader tutorial](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/upload-rules/tutorial.html), to make it easy)
  2. Try to space them out a bit, so that you can see that the dropdown from the Saved Rule Selector sorts them starting with the most recent.

## License
